src/pages/partners/_premium_agent.module.scss

```
@@ -0,0 +1,85 @@
+@use 'features/styles/theme/theme-mixins' as *;
+@import 'features/styles/theme/variables';
+
+
+.wrap_container {
+  display: flex;
+  margin-bottom: 6rem;
+  gap: 0;
+  width: 90% !important;
+  margin-inline: auto;
+  justify-content: center;
+
+
+  @media (max-width: 1024px ) {
+    flex-wrap: wrap;
+
+  }
+
+  @media (max-width: 600px ) {
+    flex-wrap: wrap;
+    justify-content: start;
+
+  }
+
+    @include breakpoints (laptop) {
+      gap: 3rem;
+      padding: 40px 0 36px;
+    }
+
+
+  }
+
+  .wrapper {
+    width: 100%;
+    max-width: 56rem;
+  }
+
+  .premium-container {
+    @include breakpoints(tablet) {
+      padding: 40px 0 36px;
+    }
+  }
+
+  .array_wrapper {
+    display: grid;
+    gap: 3rem;
+    width: 296px;
+    grid-template-columns: repeat(2, 1fr);
+    grid-template-rows: repeat(5, 1fr);
+    margin-top: 2.5rem;
```

This behaviour can be reached by common flex and inline-size/flex-basis for specific @media

At the end it may looks like 
```suggestion
    inline-size: 29.6rem;
    
     @media (min-width: 600px ) {
          inline-size: 54rem;
    }
```

We need your point of view @mohsen-deriv 🙏